.cmake-format

```diff
@@ -0,0 +1,277 @@
+# ----------------------------------
```

Comment:
I don't have a strong opinion on this, but perhaps we could hide this file within `cmake` sub-dir...? It wouldn't be aligned with the `.clang-format` file, so just to your consideration...

Reply:
As you wrote - it would be not aligned with .clang-format so I prefer to leave it here